glide.yaml

```diff
@@ -114,9 +118,8 @@ import:
     version: ^1.2.0
 
   # NB(r): make sure to use the master commit for vellum
-  # once all upstream changes are complete in github.com/couchbase/vellum.
-  - package: github.com/couchbase/vellum
-    repo: https://github.com/m3db/vellum
+  # once all upstream changes are complete in github.com/m3db/vellum.
+  - package: github.com/m3db/vellum
```

Comment:
Do we have to update all the import paths now?

Reply:
Yeah, I had to do this to be able to pull it into our internal monorepo which uses dep :/